static/app/components/search/sources/formSource.tsx

```diff
@@ -65,9 +66,15 @@ class FormSource extends Component<Props, State> {
     const {searchMap, query, params, children} = this.props;
     const {fuzzy} = this.state;
 
+    const isCustomerDomain = !!ConfigStore.get('customerDomain');
+
     const results =
       fuzzy?.search(query).map<Result>(value => {
         const {item, ...rest} = value;
+        if (isCustomerDomain && item.route === '/settings/:orgId/') {
+          item.route = '/settings/organization/:orgId/';
```

Comment:
I don't remember there being a route for `/settings/organization/:orgId`.  Using this route in the UI results in a redirect to `/settings/organization/projects/:orgId/` which triggers a bunch of 403 errors as requests are made to the slug `organization`.

I think instead you want a redirect target of `/settings/organization/` or `/settings/:orgId/`

Reply:
for some reason `/settings/organization/#anchor-tag` is being normalized to just `/settings/` however `/settings/organization/:orgId/` does get normalized to the right place. This was also the fix @roggenkemper came up with in https://github.com/getsentry/sentry/pull/73697#pullrequestreview-2155087143